src/Abp/BackgroundJobs/IBackgroundJobConfiguration.cs

```diff
@@ -15,11 +16,17 @@ public interface IBackgroundJobConfiguration
         /// <summary>
         /// Period in milliseconds.
         /// </summary>
+        [Obsolete("Use UserTokenExpirationFrequency instead.")]
         int? CleanUserTokenPeriod { get; set; }
 
+        /// <summary>
+        /// Frequency for user token expiration
```

Comment:
- Frequency is the inverse of period. This value is the period.
- End with a full stop for consistency.

Reply:
👍